components/BottomNavigation/tests/snapshot/MDCBottomNavigationItemViewSnapshotTests.m

```diff
@@ -65,154 +68,159 @@ - (void)setUp {
   self.itemView.title = kLongTitleLatin;
   self.itemView.contentHorizontalMargin = kContentHorizontalMargin;
   self.itemView.backgroundColor = UIColor.whiteColor;
-  self.itemView.badgeValue = kBadgeTitleLatin;
+  self.itemView.badgeValue = kBadgeTitleEmpty;
 }
 
 - (void)generateAndVerifySnapshot {
   UIView *backgroundView = [self.itemView mdc_addToBackgroundView];
   [self snapshotVerifyView:backgroundView];
 }
 
-- (void)changeToRTLAndArabic {
+- (void)changeToRTLAndArabicWithBadgeValue:(NSString *)badgeValue {
   if (@available(iOS 9.0, *)) {
     self.itemView.semanticContentAttribute = UISemanticContentAttributeForceRightToLeft;
   }
   self.itemView.title = kLongTitleArabic;
-  self.itemView.badgeValue = kBadgeTitleArabic;
+  self.itemView.badgeValue = badgeValue;
 }
 
 #pragma mark - Varied widths
 
-- (void)testNarrowWidthTypicalHeightLongTitleStackedLTR {
+- (void)testNarrowWidthTypicalHeightLongTitleEmptyBadgeStackedLTR {
   // When
   self.itemView.titleBelowIcon = YES;
   self.itemView.frame = CGRectMake(0, 0, kWidthNarrrow, kHeightTypical);
```

Comment:
How do you choose typical/narrow/short for size?

Reply:
Are you asking how I got the values?
They're from [the Material Guidelines article for Bottom Navigation](https://material.io/design/components/bottom-navigation.html#specs).  The "narrow" width is below minimum because it is 320 / 5, which is an older iPhone in landscape with 5 items.

Or are you asking how I choose the combination for each test? I believe I varied them so cover each combination of width + height (or at least most).